raiden-ts/src/raiden.ts

```diff
@@ -972,11 +972,29 @@ export class Raiden {
     const { signer } = chooseOnchainAccount(this.deps, subkey ?? this.config.subkey);
     // Mint token
     const customTokenContract = CustomTokenFactory.connect(token, signer);
-    const tx = await customTokenContract.functions.mint(decode(UInt(32), amount));
-    const receipt = await tx.wait();
+
+    let tx;
+    try {
+      tx = await customTokenContract.functions.mint(decode(UInt(32), amount));
+    } catch (err) {
+      this.log.error('Error sending approve tx', err);
+      throw err;
+    }
+    this.log.debug(`sent mint tx "${tx.hash}" to "${token}"`);
+
+    let receipt;
+    try {
+      receipt = await tx.wait();
+    } catch (err) {
+      this.log.error('Error mining mint tx', err);
+      throw err;
```

Comment:
Can you throw `RDN_MINT_FAILED` here so we can show the user in case of errors the same message as in the case below?